wherehows-web/app/styles/components/application/_all.scss

```diff
@@ -0,0 +1 @@
+@import 'banner-alerts';
```

Comment:
Same comment as below on that semantic meaning of using the application namespace...